src/main/java/net/sf/jabref/gui/keyboard/KeyBindingCategory.java

```diff
@@ -23,7 +23,7 @@
 
 
     private KeyBindingCategory(String name) {
-        this.name = name;
+        this.name = name.replace("&", "");
```

Comment:
Yes I know, but one may add keyboard shortcuts for actions which are not present in the menu or reorder the categories a bit.